### PR TITLE
Update admin page

### PR DESCRIPTION
**Pull request by WillDenton1989**
https://trello.com/c/4JyREeyL/69-make-admin-games-page-look-a-little-better

### CODE REVIEW

File: app/controllers/admin/games_controller.rb
Comment (WillDenton1989) [inline app/controllers/admin/games_controller.rb:1]: I tried just adding the sort_by to the end of line 7 but that failed a test so i just sorted games independently.

File: app/controllers/admin/games_controller.rb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 class Admin::GamesController < Admin::BaseController
+  layout "admin"
```
Comment (mpanetta) [inline app/controllers/admin/games_controller.rb:2]: I'd probably put the layout in the `Admin::BaseController`

File: config/locales/ux.en.yml [hunk 2]
```diff
@@ -6,16 +6,22 @@ en:
   admin:
     games:
       index:
-        header: "Curent Running Games"
+        header: "Current Running Games"
```
Comment (mpanetta) [inline config/locales/ux.en.yml:9]: Woohoo!

File: app/controllers/admin/games_controller.rb [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
+
   def index
     result = GamesService.get_games
 
-    @games = AdminGameDecorator.decorate_collection(result.games)
+    @games = AdminGameDecorator.decorate_collection(result.games).sort_by!(&:number)
```
Comment (mpanetta) [inline app/controllers/admin/games_controller.rb:7]: You will want to pay attention the the class returned by the original line, and the class returned when you add the `sort_by!` to the end.

**Pull request merged**